libs/logic-apps-shared/src/utils/src/lib/helpers/color.ts

```diff
@@ -171,3 +171,34 @@ export function getContrastTextColor(hexColor?: string): string | undefined {
 
   return luminance > 0.5 ? '#000000' : '#FFFFFF';
 }
+
+export function getAdjustedBackgroundColor(hexColor: string, isDark: boolean): string {
```

Comment:
Can you post some example screenshots of this?  I can't tell if I'm seeing it right in the gif

Reply:
haha this was me just playing around with the background color of the cards to be based on brand color, which is currently hidden behind Agents kind - I'll probably remove this, and just darken the trigger to make it different than the standard background to note that it can't be deleted